README.md

```diff
@@ -234,6 +234,34 @@ For other errors, please [open an issue](https://github.com/BlueBrain/CoreNeuron
 
 ## Developer Build
 
+#### Running Unit and Integration Tests
+
+As **CoreNEURON** is mostly used as a compute library of **NEURON** it needs to be incorporated with **NEURON** to test most of its functionality. Consequently its tests are included in the NEURON repository. To enable and run all the tests of **CoreNEURON** you need to add the `-DNRN_ENABLE_TESTS=ON` CMake flag in **NEURON**.
+Those tests include:
+* Unit tests of NEURON
+* Unit tests of CoreNEURON
+* Integration tests of NEURON
+* Integration tests of CoreNEURON
+* Integration tests between NEURON and CoreNEURON
+* [ringtest](https://github.com/neuronsimulator/ringtest) tests with NEURON and CoreNEURON
+* [testcorenrn](https://github.com/neuronsimulator/testcorenrn) tests with NEURON and CoreNEURON
+Some of those tests are going to be also run with various backends in case that those are enabled (for example with GPUs).
+To build NEURON with CoreNEURON and run the tests you need to:
+
+```bash
+cd nrn/build
+cmake .. \
+  -DNRN_ENABLE_CORENEURON=ON \
+  -DNRN_ENABLE_INTERVIEWS=OFF \
+  -DNRN_ENABLE_RX3D=OFF \
+  -DCMAKE_INSTALL_PREFIX=$HOME/install
+  -DCMAKE_C_COMPILER=icc \
+  -DCMAKE_CXX_COMPILER=icpc \
+  -DNRN_ENABLE_TESTS=ON
+make -j
```

Comment:
```suggestion
cmake --build . --parallel
```

Reply:
Should this be changed in other places as well?

Reply:
no strong opinion - until now everywhere we have make. (if this is preferred way and portable with cmake versions we support, 3.8, then ok to change?)

Reply:
This is supported in `cmake 3.8` as well. If you think it's fine maybe we should change it everywhere in CoreNEURON and in NEURON as well to be consistent